media/js/firefox/new/experiment-firefox-new-waitface.js

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+
+/* Experiment: https://bugzilla.mozilla.org/show_bug.cgi?id=1434764
+ * URL: https://www.mozilla.org/en-US/firefox/new/?xv=waitface
+ * Audience: Desktop, Non-Fx, en-US with utm_medium=display&utm_source=doubleclick    
+ * Traffic: 100% audience, 50/50 split*/
+
+(function() {
+    'use strict';
+
+    var isDesktop = function() {
+        return ! /android|ios/.test(site.platform);
+    };
+
+    var notFirefox = function () {
+        return ! /\sFirefox/.test(navigator.userAgent);
+    };
+
+    var paramsMatch = function() {
```

Comment:
Likewise here we also have a [search param library](https://github.com/mozilla/bedrock/blob/master/media/js/base/search-params.js) in our common bundle that makes it easy to get URL params. That may be helpful here.